onnxruntime/python/tools/transformers/onnx_model.py

```diff
@@ -426,6 +426,54 @@ def find_first_child_by_type(self, node, child_type, input_name_to_nodes=None, r
 
         return None
 
+    def match_child_path(
+        self,
+        node,
+        child_op_types,
+        child_output_index=None,
+        return_indice=None,
+        exclude=[],  # noqa: B006
```

Comment:
```suggestion
        exclude=None,
```

https://docs.astral.sh/ruff/rules/mutable-argument-default/